### PR TITLE
Fixed Maven warnings for deprecated org.junit.Assert.assertThat(...)

### PR DESCRIPTION
**Pull request by dmaidaniuk**
See warn: <T>assertThat(T,org.hamcrest.Matcher<? super T>) in org.junit.Assert has been deprecated

### CODE REVIEW

**Comment by dmaidaniuk** [comment]
Hi @chkal, @erdlet ,
Could you please take a look at my clean-up PR? I fixed Maven warnings about the usage of deprecated JUnit methods `org.junit.Assert.assertThat(...)`.

**Review by erdlet — APPROVED** [review]
Looks good. I'm just not sure if those additional `assertNotNull` checks clutter up the code🤔

File: core/src/test/java/org/eclipse/krazo/util/AnnotationUtilsTest.java [hunk 1]
```diff
@@ -69,18 +68,21 @@ public void hasAnnotation() {
     @Test
     public void getAnnotationOnMethod() throws NoSuchMethodException {
         View view = AnnotationUtils.getAnnotation(someController.getClass().getMethod("start"), View.class);
-        assertThat(view.value(), is("start.jsp"));
+        assertNotNull(view.value());
```
[the remark this answers is not in the document]
Comment (dmaidaniuk) [reply core/src/test/java/org/eclipse/krazo/util/AnnotationUtilsTest.java]: @erdlet Yes, I agree that these checks for NPE aren't necessary here. Updated PR.

[the remark this answers is not in the document]
Comment (erdlet) [reply core/src/test/java/org/eclipse/krazo/util/AnnotationUtilsTest.java]: Thanks 👍

**Comment by chkal** [comment]
@dmaidaniuk Although I'm a bit late: Thanks and good work! :+1:

**Pull request merged**